dipy/reconst/dki.py

```diff
@@ -2530,13 +2647,20 @@ def ls_fit_dki(
         Inverse of the design matrix.
     return_S0_hat : bool, optional
         Boolean to return (True) or not (False) the S0 values for the fit.
-    weights : bool, optional
-        Parameter indicating whether weights are used.
+    weights : array ([X, Y, Z, ...], g), optional
```

Comment:
But I don't want the default to be None, I want it to be False, so that when weights is False it will calculate OLS instead of WLS. Is it really necessary to risk breaking this PR just so it can be None instead of False?